cmd/account/cli.go

```diff
@@ -104,6 +117,19 @@ func (o *cliOptions) run() error {
 
 	ctx := context.TODO()
 	var accountID string
+	if o.clusterID != "" {
+		accountClaim, err := k8s.GetAccountClaimFromClusterID(ctx, o.kubeCli, o.clusterID)
+		if err != nil {
+			return err
+		}
+		if accountClaim == nil {
```

Comment:
Wouldn't this be covered by the err? `ErrNotFound` or something similar?

Reply:
In this case, no, because GetAccountClaimFromClusterID has to use the k8s List function, and not Get.  While Get will throw an error of ErrNotFound or whatever it is, List will just return an empty list.